src/components/icons/iconUtils.ts

```diff
@@ -11,7 +11,7 @@ export const getSizeClass = (size: 's' | 'm' | 'l') => {
 	}
 };
 
-export const getColorClass = (color: 'base' | 'white' | 'primary' | 'secondary' | 'error') => {
+export const getColorClass = (color: 'base' | 'white' | 'primary' | 'secondary' | 'error' | undefined) => {
```

Comment:
Why do we have and undefined, color class?

Reply:
@malinovic fixed